docs/pages/lookback_zscore.md

```diff
@@ -0,0 +1,94 @@
+# The "lookback z-score method" of benchmark result analysis
+
+A core value of Conbench is the ability to compare incoming benchmark results with historic results to make data-driven decisions about performance.
+The primary method of doing this in Conbench is the "lookback z-score" analysis.
+This page describes how that process works.
+
+## Overview
+
+The lookback z-score analysis assigns a numeric z-score value to a benchmark result that represents how "good" the result is compared to a historic distribution of similar results.
+Coupled with a threshold, this can be used to determine whether the benchmark result is indicative of a performance regression or improvement.
+
+## Choosing the historic distribution of results
+
+When scoring a contender benchmark result, in order to control for variables that don't have to do with actual performance changes, each result in the historic distribution must share the contender's:
+
+- repository
+- hardware
+- case permutation
+- context
+
+Results with errors will not be included, even if they also have data.
+
+Additionally, when using the lookback z-score method, one must provide a baseline commit (or baseline result or baseline run, which is ultimately associated with a baseline commit).
+The historic distribution will be drawn from all runs on commits in the baseline commit's git ancestry, up to and including all runs on the baseline commit itself.
+The maximum number of commits used is defined by the Conbench server's `DISTRIBUTION_COMMITS` parameter, which defaults to 100.
+In certain cases, there will be fewer than this number: if there are not that many commits in the ancestry, or if a distribution change was annotated by someone acting as a data curator -- see below for more details.
+
+In some cases (e.g. navigating the UI, or generating alerts with the `benchalerts` library), Conbench will provide the user with a sensible baseline commit.
```

Comment:
Should we link to https://conbench.github.io/conbench/pages/benchmarking_and_the_dev_process.html somewhere here?

Reply:
Done!